src/layer/form-widgets/index.md

```diff
@@ -0,0 +1,305 @@
+
+### Value Map
+
+:::tip Example project available
+This public project <MerginMapsProject id="documentation/form-widgets" /> contains a Value Map field. Download or clone it to see the setup.
+:::
+
+1. Right-click on a layer, select **Properties** and go to the **Attributes form** tab.
+2. In the list of **Available Widgets** select the field you want to work with (here: `valuemap`)
+3. In the **Widget Type** tab, select the **Value Map** option from the drop-down menu and fill in the table below. 
+   **Value** is what will be stored in the field (these can be coded values or shortened names, here we use `1`, `2`, `3`). **Description** is what will be displayed in the form and in the attributes table (here: `value 1`, `value 2`, `value 3`).
+4. **Apply** the changes. Don't forget to save and sync your project!
+
+![QGIS value map field form](./qgis-form-valuemap.jpg "QGIS value map field form")
+
+Now you can select the value from a drop-down menu in the <MobileAppNameShort />:
+![Mergin Maps mobile app value map field form](./mobile-form-valuemap.jpg "Mergin Maps mobile app value map field form")
+
+
+### Value relation
+
+:::tip Example project available
+This public project <MerginMapsProject id="documentation/form-widgets" /> contains a Value Relation setup. Download or clone it to explore it in more detail.
+:::
+
+The **Value Relation** widget is similar to the [Value Map](#value-map) tool, but the values for the drop-down menu come from another non-spatial table.
+
+There are some benefits of using the Value Relation widget, such as:
+  - Option to select multiple values from the list (in this case, the field needs to be of `text` data type).
+  - The table that contains values can be edited in the field. For example, if you have missed a value in your list for the drop-down menu, you can edit the table in the <MobileAppNameShort />, add the value and use it during your survey. See [Working with non-spatial tables](../non-spatial-data/) for more details.
+  - Searching the values: if you have a large list of values, it will become cumbersome to find the right value. With Value Relation widget, you will have to option to search for values in the list in the <MobileAppNameShort />.
+
+
+To set up **Value Relation** in QGIS:
+
+1. Right-click on a layer, select **Properties** and go to the **Attributes form** tab.
+2. In the list of **Available Widgets** select the field you want to work with (here: `value-relation-multi-select`)
+3. In the **Widget Type** tab, select **Value Relation** from the drop-down menu and set it up as follows:
+   - Select the **Layer** that contains the values (here: `value-relation-table`)
+   - **Key column** is the field that contains the values (here: `fid`)
```

Comment:
We should advice people not to use `fid` columns for references. The thing is they might change during the sync process. This would happen if two people add a new entry to the same layer and then they sync. One of the features will have a different `fid` then which might result in referencing a different feature in VR than anticipated.

There are basically two options: (1) one is to generate `uuid` for each feature to use instead of the `fid` column, or (2) use the same column as for `value`. However, if someone uses the same column, they must ensure that the values will be unique.